appveyor.yml

```diff
@@ -7,6 +7,8 @@ environment:
       PYTHON_ARCH: "32"
       MINICONDA: C:\Miniconda35
       DATALAD_TESTS_SSH: 1
+      DATALAD_LOG_LEVEL: 5
+      DATALAD_LOG_TRACEBACK: "collide"

```

Comment:
Why those? Travis doesn't do that.

Reply:
Left over. Will remove alongside skipping the hanging test on windows (if we agree on that).